src/renderer/mod.rs

```diff
@@ -175,6 +175,7 @@ impl Renderer {
             self.surface = None;
         }
 
+        #[allow(deprecated)]
```

Comment:
Don't do this.

Reply:
I'm curious about why this was added. Any details @uhuntu?

Reply:
When I commit something, it will say
```
error: use of deprecated item 'skia_safe::core::types::Budgeted::YES': use Yes
   --> src/renderer/mod.rs:180:28
    |
180 |             let budgeted = Budgeted::YES;
    |                            ^^^^^^^^^^^^^
    |
    = note: `-D deprecated` implied by `-D warnings`

error: aborting due to previous error

error: could not compile `neovide`.
```

Reply:
`...: use Yes`

use `Budgeted::Yes` to omit the warning.